nni/nas/oneshot/pytorch/supermodule/differentiable.py

```diff
@@ -376,6 +404,16 @@ def export(self, memo):
             result[name] = spec.values[chosen_index]
         return result
 
+    def export_probs(self, memo):
```

Comment:
seems the `export_probs` logic are similar? and some have type hints and some not, anyway, it's not matter.

Reply:
Yes. You might notice all DifferentiableXXX are very similar, not only `export_probs`. The framework design is to blame.